setup.py

```diff
@@ -115,7 +115,8 @@ def run_tests(self):
         'pyyaml',
         'argparse',
         'argcomplete',
-        'future'
+        'future',
+        'pyBarcode==0.8b1'
```

Comment:
Another option might be [steenzout](https://github.com/steenzout/python-barcode). It seems to have a bit more traction. @MicroJoe, could you maybe check both projects and decide which one is better?